### ISSUE TITLE
Python: `index_summary` : Cluster Count == # of seq * 2 of a PE run?

### ISSUE THREAD

**Issue by sklages** [Illumina/interop#271]
I haven't noticed before,

`index_summary(run_metrics, level='Barcode')` returns as `Cluster Count` the total number of reads of a *PE* run, not the acutal number of clusters (= single reads, R1 seq count, whatever).
That's confusing ..

Is that intented?

This is `interop-1.1.23`.

**Comment by ezralanglois** [comment]
I think you mean a dual index run, and yes, this is annoying. As far as I know, it has always been this way and we will need to rev the major minor version because this potentially will be a breaking change for downstream applications.

**Comment by sklages** [comment]
No, simple **paired end** run, no matter if single index, dual index run. `Cluster Count` always seems to be the sum of sequences of `R1` and `R2`... 

Upper is the actual `R1` fastq file, the lower is the outpzt from `index_summary(run_metrics, level='Barcode')`.
![image](https://user-images.githubusercontent.com/1837422/118944525-eed10800-b954-11eb-95cd-0d4bc4100ac0.png)

**Comment by ezralanglois** [comment]
Ya, you are right. I misinterpreted the code.
